### ISSUE TITLE
pull deis org via command line flag

### ISSUE THREAD

**Issue by jchauncey** [deis/deisrel#27]
see conversation here - https://github.com/deis/deisrel/pull/26

**Comment by bacongobbler** [comment]
for clarity, right now in several places in the `action` package we hardcode `deis` as the organization name. We should provide a `--organization` global flag to codegangsta/cli so that we can use this against other organizations. `deisrel --organization=bacongobbler git tag` could be useful for contributors.

**Comment by jchauncey** [comment]
yeah i could see us making this a more useful tool for releasing open source projects in general

**Comment by arschles** [comment]
we'd probably also want to parameterize the repository names. that work should take place in this issue as well.

**Comment by bacongobbler** [comment]
this is nice to have but not a requirement for rc1.

**Comment by Cryptophobia** [comment]
This issue was moved to teamhephy/deisrel#6
